cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -21,6 +21,14 @@ var strings = require( '../modules/util/strings' );
  */
 function Multiselect( element ) { // eslint-disable-line max-statements, inline-comments, max-len
 
+  var BASE_CLASS = 'cf-multi-select';
+
+  // TODO: Add 'cf-multi-select' class to <select> menu markup
```

Comment:
Any reason not to update them in this PR? https://github.com/cfpb/cfgov-refresh/blob/e1ed9c3ae5e1b5815b0615a8caea8ab796ae0ed2/cfgov/v1/forms.py#L81-L88